catalog/app/utils/json-schema/json-schema.js

```diff
@@ -126,7 +126,7 @@ export function makeSchemaDefaultsSetter(optSchema) {
     const validate = ajv.compile(schema)
 
     return (obj) => {
-      validate(obj)
+      validate(R.clone(obj))
```

Comment:
i thought the point of this call is to mutate the object and set the defaults -- in this case i believe you should return the cloned object

Reply:
I think, I made a mistake, will investigate it. Thank you